### DISCUSSION TITLE
The one thing I hope that happens with this mod

### DISCUSSION THREAD

**Discussion by MarzkYT** [gennariarmando/v-hud#100] [General]
It is a excellent mod the only thing that I dislike that it don’t support enb. 
please make it support enb graphic mods.
I don’t care how long it takes

**Comment by gennariarmando** [comment]
It does already

**Reply by ruguysgoingtrickortreating** [reply]
@gennariarmando sorry for reviving an old thread but how? I'm trying to get the gta v enb to work but it launches to a black screen.
